support-frontend/assets/components/headers/header/header.tsx

```diff
@@ -104,13 +73,9 @@ export default class Header extends Component<PropTypes, State> {
 			containerRef
 		) {
 			this.observer = new window.ResizeObserver(() => {
-				this.setState(
-					getMenuStateMetrics({
-						menuRef,
-						logoRef,
-						containerRef,
-					}),
-				);
+				this.setState({
+					isTestUser: getGlobal<boolean>('isTestUser'),
+				});
```

Comment:
Is it important to set this on window resize, or could it be set once earlier in the process?

Reply:
Good point, we wouldn't need to wait to call `setState` on `ResizeObserver` now we're not using JS to adjust the menu items.

Reply:
Perfect, thanks for making that change!